extension-manifest-v2/app/trackers-preview/trackers-preview.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import {
+	getStats,
+	close,
+	disable,
+	updateIframeHeight
+} from '@whotracksme/webextension-packages/packages/trackers-preview/src/page_scripts';
+
+import { define, html } from 'hybrids';
+
+const domain = new URLSearchParams(window.location.search).get('domain');
+const stats = getStats(domain);
```

Comment:
Can we skip the call here if domain is "null"? Maybe it works as is, but getStats is not expected to deal with nullable values (and arguably should not need to): https://github.com/whotracksme/webextension-packages/blob/main/packages/trackers-preview/src/page_scripts/index.js#L12

Also, wouldn't `window.location` be always the search page here?

Reply:
It cannot be null, as the iframe is only loaded by the content script, which runs on google search results, and generates a link to it with a target result domain.